app/src/main/res/layout/welcome.xml

```diff
@@ -42,13 +42,26 @@
       app:layout_constraintBottom_toBottomOf="@id/welcome_text_view"
       app:layout_constraintTop_toTopOf="@+id/welcome_text_view" />
 
+    <TextView
```

Comment:
Suggesting to set up [macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) which helps in reformat and rearrange the code.